cluster/doc.go

```diff
@@ -102,5 +102,58 @@ You can configure the frequency of the reconnection attempts using config.Cluste
 	r.Jitter = 0.0
 	r.Multiplier = 1.05
 
+# TLS/SSL
+
+Hazelcast supports encrypted communication between the client and the server using TLS/SSL protocol with key stores and trust stores.
+This feature requires Hazelcast Enterprise.
+
+In order to use TLS encryption, the Hazelcast members should be configured.
+See https://docs.hazelcast.com/hazelcast/latest/security/tls-ssl.html#tlsssl-for-hazelcast-members for more information.
+
+In order to enable TLS on the client side, set config.Cluster.Network.SSl.Enabled to true.
+
+	// error handling is omitted for brevity.
+	var config hazelcast.Config
+	config.Cluster.Network.SSL.Enabled = true
+
+If you need to set a custom tls.Config, for example to turn off verification, you need to assign the tls.Config before any other SSL configuration:
+Check out this page for further details about tls.Config options: https://pkg.go.dev/crypto/tls#Config
+
+	// warning: never set InsecureSkipVerify: true on production
+	// config.Cluster.Network.SSL.SetTLSConfig(&tls.Config{InsecureSkipVerify: true})
+
+You need to set the host name with:
```

Comment:
Why user needs to set the hostname here? Why do we say "need" instead of "can"? Is it also related with the paragraph above, I did not get it .

Reply:
Otherwise Go std lib returns an error that it must be set.